deploy/sre-prometheus/management-cluster/100-dynatrace-workloads-monitoring.PrometheusRule.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: monitoring.rhobs/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-dynatrace-workloads-failing
+    role: alert-rules
+  name: sre-dynatrace-workloads-failing
+  namespace: openshift-observability-dynatrace
+spec:
+  groups:
+  - name: sre-dynatrace-workloads-failing
+    rules:
+    - alert: DynatraceMonitoringStackDownSRE
+      expr: |
+          sum(kube_node_status_condition{condition="Ready", status="false"}) == 0 and
```

Comment:
why we need this condition?

Reply:
Hey Haoran
There was a concern what happens if the node is not ready on Day 2. But I think you are right. We don't need this query anymore. This was part of the initial query and might have been crept up here. If day 1 rollout fails, we are not relying on any node anyway. At that time we are just concerned about the service availability which is already covered.

I will remove the condition